envoy/router/BUILD

```diff
@@ -144,3 +146,25 @@ envoy_cc_library(
         "//envoy/server:factory_context_interface",
     ],
 )
+
+envoy_cc_library(
+    name = "router_path_match_policy_interface",
```

Comment:
nit: this library name should match the file name so something like "path_matcher_interface" (and below)